CHANGELOG.md

```diff
@@ -15,9 +15,45 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [3.0.0-rc00] - 2024-06-11
 
+This release candidate is compatible with [#node RC2](https://github.com/casper-network/casper-node/tree/release-2.0.0-rc2)
+
 ## Added
 
-- Condor functionalities added
+- `CLAccountHash` now has `toFormattedStr()` and `fromFormattedStr(hexStr: string)` methods
+- Changed `enum KeyVariant` to `enum KeyTag`, backfilled all the variants that were missing, added condor-specifin ones
+- `StoredValue` has new fields which represent enum variants and are reuqired for 2.x compatibility:
+  - `NamedKey`
+  - `AddressableEntity`
+  - `BidKind`
+  - `Package`
+  - `ByteCode`
+  - `MessageTopic`
+  - `Message`
+  - `Reservation`
+  - `EntryPoint`
+- `TransactionUtil` class with all the subclasses to support sending Transaction to the node
+- `transaction`, `getAccountInfo`, `queryBalanceDetails`, `getTransactionInfo`, `waitForTransaction`, `queryGlobalState` methods added to `CasperServiceByJsonRPC` class
+- `Bid` class has new fields: `validator_public_key`, `vesting_schedule`
+- `merkleProof` field to `EraSummary` class
+- optional `latest_switch_block_hash` added to `GetStatusResult`
+
+## Breaking changes compared to 2.x
```

Comment:
I know this is special situation but I would follow the https://keepachangelog.com/en/1.1.0/ convention so this probably should be just `## Changed` and then in description we can add that its compared to previous stable version which was 2.x. 

I would do this because if we will want to generate some documents from this CHANGELOG file it should make it easier.